### PR TITLE
#9 Task. Alexander Zenchyk

### PR DESCRIPTION
**Pull request by zeeenjaaa**
https://travis-ci.com/github/zeeenjaaa/js-assignments/builds/178242316

### CODE REVIEW

File: task/09-functions-n-closures-tasks.js [hunk 1]
```diff
@@ -104,7 +115,17 @@ function memoize(func) {
...
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return function() {
+        for (let i = 0; i <= attempts; i++){
+            try {
+                return func();
+            } catch (error) {}
+
+            if(i == attempts){
```
Comment (vramaniuk) [inline task/09-functions-n-closures-tasks.js]: "operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by zeeenjaaa** [comment]
Thank you for review and good lesson and time!
I've fixed all issues.

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**